src/components/LanguageSelector/LanguageSelectorButton.tsx

```diff
@@ -27,8 +25,6 @@ export const LanguageSelectorButton = ({
     <button
       data-testid="languageSelectorButton"
       className={classes}
-      aria-expanded={isOpen}
-      aria-controls="language-options"
```

Comment:
ℹ️ These are now passed in as props from `LanguageSelectorDropdown` since that's the only context they need applied